py-polars/polars/_utils/parse_expr_input.py

```diff
@@ -121,6 +121,12 @@ def parse_as_expression(
     elif isinstance(input, list) and not list_as_lit:
         expr = F.lit(pl.Series(input), dtype=dtype)
         structify = False
+    elif isinstance(input, pl.DataFrame):
```

Comment:
This is not the right place to add this. It should be added in the `lit` function.